src/components/Charts/ChartVaccinations.tsx

```diff
@@ -1,37 +1,245 @@
+import React, { useCallback, Fragment } from 'react';
+import moment from 'moment';
+import { isNumber } from 'lodash';
+import { AxisLeft, AxisBottom } from '@vx/axis';
+import { Group } from '@vx/group';
+import { ParentSize } from '@vx/responsive';
+import { scaleUtc, scaleLinear } from '@vx/scale';
+import { useTooltip } from '@vx/tooltip';
+import { formatPercent, formatUtcDate } from 'common/utils';
 import { Column } from 'common/models/Projection';
-import { formatPercent } from 'common/utils';
-import { getMetricDefinition, Metric } from 'common/metric';
+import { Tooltip, RectClipGroup } from 'components/Charts';
+import { ScreenshotReady } from 'components/Screenshot';
+import GridLines from 'components/Explore/GridLines';
+import * as Styles from 'components/Explore/Explore.style';
+import { Series } from 'components/Explore/interfaces';
+import ChartSeries, { SeriesMarker } from 'components/Explore/SeriesChart';
+import ChartOverlay from 'components/Explore/ChartOverlay';
+import { findPointByDate, getTimeAxisTicks } from 'components/Explore/utils';
+import * as ChartStyle from './Charts.style';
 
-const CAP_Y = 1;
+const getDate = (d: Column) => new Date(d.x);
+const getY = (d: Column) => d.y;
 
-const getPointText = (valueY: number) => formatPercent(valueY, 1);
+const xTickFormat = (date: Date, isMobile: boolean) => {
+  const momentDate = moment(date);
+  // Shows the year if the tick is in January (0) or December (11)
+  const dateFormat =
+    momentDate.month() === 0 || momentDate.month() === 11 ? 'MMM YYYY' : 'MMM';
+  return momentDate.format(dateFormat);
+};
+
+const VaccinesTooltip: React.FC<{
+  date: Date;
+  seriesList: Series[];
+  left: (d: Column) => number;
+  top: (d: Column) => number;
+  subtext: string;
+}> = ({ seriesList, left, top, date }) => {
+  const [seriesInitiated, seriesCompleted] = seriesList;
+  const pointCompleted = findPointByDate(seriesCompleted.data, date);
+  const pointInitiated = findPointByDate(seriesInitiated.data, date);
+
+  return pointCompleted && pointInitiated ? (
+    <Tooltip
+      width={'160px'}
+      top={top(pointInitiated)}
+      left={left(pointCompleted)}
+      title={formatUtcDate(new Date(pointCompleted.x), 'MMM D, YYYY')}
+    >
+      <Styles.TooltipLine>
+        <Styles.TooltipLabel>
+          {seriesInitiated.tooltipLabel}
+        </Styles.TooltipLabel>
+        <Styles.TooltipValue>
+          {isNumber(pointInitiated.y)
+            ? formatPercent(pointInitiated.y, 1)
+            : '-'}
+        </Styles.TooltipValue>
+      </Styles.TooltipLine>
+      <Styles.TooltipLine>
+        <Styles.TooltipLabel>
+          {seriesCompleted.tooltipLabel}
+        </Styles.TooltipLabel>{' '}
+        <Styles.TooltipValue>
+          {isNumber(pointCompleted.y)
+            ? formatPercent(pointCompleted.y, 1)
+            : '-'}
+        </Styles.TooltipValue>
+      </Styles.TooltipLine>
+    </Tooltip>
+  ) : null;
+};
+
+const DataHoverMarkers: React.FC<{
```

Comment:
I might recommend just making this: `DataHoverMarker` and have it just take a `Series`, pushing the `map` up to the calling component.  I think it makes the code a bit more declarative and gives you more flexibility with how you display this

Reply:
That might not work in general, because sometimes the marker might have other elements. In this case, for example, we are adding a "tracker line", a vertical line that goes between the 2 points (doing that on a separate PR)